.github/workflows/master.yml

```diff
@@ -0,0 +1,87 @@
+name: Build and Release from Master
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16, 18]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+
+      - name: Install hub
+        run: |
+          sudo apt update
+          sudo apt install hub
+          hub --version
+
+      - name: Install github-changelog-generator
+        run: gem install github_changelog_generator --user-install
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Set Ribby as git user
+        run: |
+          git config user.name "Ribby"
+          git config user.email "bot@lftechnology.com"
+
+      - name: Check format
+        run: yarn format:check
+
+      - name: Lint code
+        run: yarn lint
+
+      - name: Build project
+        run: yarn build
+
+      - name: Test coverage
+        run: yarn test:coverage
+
+      - name: Upload coverage to Codecov
+        run: yarn codecov
+
+      - name: Determine branch
+        id: branch
+        run: echo "::set-output name=branch::$(if [ ${{ github.event_name }} == 'pull_request' ]; then echo ${{ github.event.pull_request.head.ref }}; else echo ${{ github.ref }}; fi)"
+
+  deploy:
```

Comment:
@sbimochan Rename this to publish. We need to publish the package which is a better name.